pysiaf/utils/tools2.py

```diff
@@ -0,0 +1,217 @@
+"""A collection of helper functions to support pysiaf calculations and functionalities
```

Comment:
This file should not be added to the project. Instead, the new function should be added to the existing `/utils/tools.py` file.